spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/test/java/com/example/SpannerRepositoryTests.java

```diff
@@ -52,7 +52,7 @@
  */
 @RunWith(SpringRunner.class)
 @ActiveProfiles("test")
-@TestPropertySource("classpath:application.properties")
+@TestPropertySource("classpath:application-test.properties")
```

Comment:
Why are you modifying this?

Reply:
The other samples actually use their own test-properties files like this. I think it should've been this way originally. 

These are test files btw.